persistance/src/main/java/cz/muni/fi/pv254/entity/User.java

```diff
@@ -18,8 +18,6 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @NotNull
-    @Column(nullable = false)
```

Comment:
discussion needed - algorithms are working based on user, they are using steam id for user searching, either generate random and check if exists, or add option in algorithms to search by our id, not steam id

Another problem - if we will store anonym user in our database, it can influence CF algorithm in wrong way